demisto_sdk/commands/lint/linter.py

```diff
@@ -1306,3 +1337,280 @@ def _get_commands_list(self, script_obj: dict):
+            latest_native_image_full_name = f"{NATIVE_IMAGE}:{latest_tag}"
+
+        else:  # latest tag not found
+            err_msg = (
+                f"{log_prompt} - {script_id} - Error: Failed getting the native image latest tag from"
+                f" Docker Hub."
+            )
+            logger.error(err_msg)
+            raise Exception(err_msg)
+
+        return latest_native_image_full_name
+
+    def _get_specific_native_image(
+        self,
+        native_image: str,
+        native_image_config: NativeImageConfig,
+        supported_native_images: Set[str],
+        script_id: str,
+    ) -> Union[str, None]:
+        """
+        Gets a specific native image name, and finds it's reference (tag) in the docker_native_image_config.json
+        file. If the native image name doesn't exist in the file or the integration/script that lint runs on doesn't
+        support this native image - write to the logs that the checks on docker will be skipped.
+        Args:
+            native_image (str): Name of the Native image to run on.
+            native_image_config (NativeImageConfig): NativeImageConfig obj based on the docker_native_image_config.json.
+            supported_native_images (set): A set including the names of the native images supported for the
+                                           script/integration that lint runs on.
+            script_id (str): The ID of the integration/script that lint runs on.
+        Returns (str): The native image reference (tag).
+        """
+        log_prompt = f"{self._pack_name} - Get Specific Native Image"
+        logger.info(f"{log_prompt} - {native_image} - Started")
+
+        if native_image not in native_image_config.native_images:
+            # Server version is invalid or not exist in the docker_native_image_config.json
+            logger.error(
+                f"{log_prompt} - Skipping checks on docker for {native_image}. The requested native image: "
+                f"{native_image} is not supported. For supported native image versions please see: "
+                f"'Tests/docker_native_image_config.json'"
+            )
+            # TODO: Maybe should throw an exception instead
+            return None
+
+        elif native_image not in supported_native_images:
+            # Integration/Script doesn't support the requested native image
+            logger.info(
+                f"{log_prompt} - Skipping checks on docker for {native_image} - {script_id} does not support the "
+                f"requested native image: {native_image}."
+            )
+            return None
+
+        else:
+            # Integration/Script supports the requested native image - find the relevant tag to run on
+            native_image_reference = native_image_config.get_native_image_reference(
+                native_image
+            )
+
+            return native_image_reference
+
+    def _get_docker_images_from_yml(self, script_obj: Dict) -> List[str]:
```

Comment:
looks like all this method does is log and then call another method. Can we log outside and delete this function?+ we should rename the get_all_docker_images instead

Reply:
yea... I know it is kind of unnecessary... I added this func mainly for the readability of the code. I didn't want to change the name and docstring of the func in tools cause we use it in the YML splitter... so I wrapped it with this func. 
I can change it if you think it would be better.

Reply:
Changed it.